docs/rules/no-nested-ternary.md

```diff
@@ -0,0 +1,20 @@
+# Disallow nested ternary expressions
+
+Nesting ternary expressions can make code more difficult to understand.
+
+One level of nesting is allowed as long as the nested ternary is wrapped in parens
+
```

Comment:
You need to mention how it's an improved version of the built-in ESLint rule (and link to it) and what it improves.

Also, at the bottom, recommend disabling the built-in rule and show how to do that.

Reply:
In the readm, you marked it as "partly fixable", so would be good to explain what is and what is not fixable.

Reply:
I tried to reword the readme. Let me know if that makes more sense